### ISSUE TITLE
rankPlayers() function gives an breaking error when there's more than one player.

### ISSUE THREAD

**Issue by leemetme** [theusaf/kahoot-session#1]
In **WSHandler.js** there is a hefty game-breaking bug that will crash the whole program. When there is more than one player, the original **rankPlayers()** code, which is listed down below, will break and give an **undefined** variable to the **[].sort()** function. This is because new players, especially when there's one or more, will not all have the **a/b.info** object, where the **totalScore** is stored because they have not answered any questions yet. This is, however, really easy to fix. All you need to do is check if the user has the **a/b.info** in the object keys, if not, add it and add a **totalScore** of 0. Pushing this fix in, I noticed another peculiar thing, the sorted list of players was the wrong way around, it was sorted ascending, instead of descending. So that meant that the scoreboard listed the player who answered the best last, and the player who answered the worst, first. This was also quite easy to fix by switching the statements around the - sign in the **return** statement.

Original code:
```
rankPlayers(){
    return JSON.parse(JSON.stringify(this.players)).sort(function(a,b){
      return a.info.totalScore - b.info.totalScore;
    });
  }
```

My version:
```
rankPlayers(){
    return JSON.parse(JSON.stringify(this.players)).sort(function(a,b){

      // a.info

      if ("info" in a){
        // Great.
      } else {
        a.info = {}
        a.info.totalScore = 0
      }

      // b.info

      if ("info" in b){
        // Great.
      } else {
        b.info = {}
        b.info.totalScore = 0
      }

      return b.info.totalScore - a.info.totalScore;
    });
  }
```

Though I was quite annoyed by this bug, I still have to thank you for this wonderful little program. 

Thanks!

**Comment by theusaf** [comment]
Thanks for finding the issue and explaining how to fix it. I'll fix it soon. Enjoy!

**Comment by theusaf** [comment]
Fixed in [d6db621](https://github.com/theusaf/kahoot-session/commit/d6db621180aab2999676e50852bb980fd59258b7)
